docs/installation.rst

```diff
@@ -0,0 +1,11 @@
+Installation
+==================
+
+SublimeLinter and corresponding linter plugins should be installed using `PackageControl <https://packagecontrol.io/installation>`_.
+
+First install PackageControl and then see its `usage <https://packagecontrol.io/docs/usage>`_,
+which explains how to install plugins to SublimeText.
+
+Notice, that it may be required to restart SublimeText after plugin installation.
```

Comment:
The comma is wrong. Or well, I certainly would not write it.

Reply:
fixed.